app.css

```diff
@@ -103,6 +104,28 @@ ul {
   width: 50%
 }
 
+.pane img.index-image {
+  width: 100%;
+}
+
+.pane.index-pane .index-media {
+  /* Align sub-items to top */
+  align-items: start;
+  display: flex;
+}
+
+.pane.index-pane .index-media .index-media-image {
+  margin-right: 16px;
+
+  /* Set the width for the media object */
+  width: 200px;
+}
+
+.pane.index-pane .index-media .index-media-content {
+  /* Take the remaining width */
+  flex: 1;
```

Comment:
Can you make this `flex-grow`, just to be more specific?

Reply:
i tried that and i thought it makes it look worse

flex: 1
![image](https://user-images.githubusercontent.com/26289/110736946-5ba38580-81e1-11eb-8431-881e8850d76d.png)

flex: flex-grow
![image](https://user-images.githubusercontent.com/26289/110736984-71b14600-81e1-11eb-83b0-af49b4a7bc36.png)

Reply:
Oh, I meant use that instead of `flex`, which is a shorthand property.

`flex-grow: 1`

Reply:
Even that has the same behavior with the images being smaller 🤔

Reply:
Maybe you need to do a hard reload? I made a PR to your branch with the change I'm talking about. `flex-grow` is actually used in other places in that file already, I noticed.

Reply:
I changed to

```css
.pane.index-pane .index-media .index-media-content {
  /* Take the remaining width */
  flex-grow: 1;
  flex-basis: 0%;
}
```

and that gives the same look as `flex: 1`